elements/link-modal.js

```diff
@@ -120,12 +120,18 @@ function createModal () {
   let link = ''
   let el = ''
 
-  const modal = Modal({ onunload, onexit, render })
+  const modal = Modal({
+    onunload,
+    onexit,
+    render,
+    class: 'modal'
+  })
 
   return modal
 
   function onexit () {
```

Comment:
can be refactored if we get a reference to the current element here, either via context or an argument:

```js
function onexit () {
  this.parentNode.removeChild(this)
}
```

or

```js
function onexit (el) {
  el.parentNode.removeChild(el)
}
```

I'd prefer the former